pcap/pcap.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2014 Damjan Cvetko. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the same directory
+// containing this source file.
```

Comment:
Is this supposed to be here?

Reply:
Yes, we should leave people's copyrights in place... I updated the comment.  In the old days, we put copyrights at the top of every source file.  Now, the convention is the LICENSE file in the root of the repo.  But we should leave this copyright here since I copied the code and conform with the LICENSE, which we do.